frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -307,3 +307,24 @@ export const getPipelineRunParams = (pipelineParams: PipelineParam[]): PipelineR
     }))
   );
 };
+
+export const convertUTC = (time?: string): Date =>
+  time ? new Date(new Date(time).toUTCString()) : new Date(new Date().toUTCString());
+
+export const pipelineRunDuration = (run: PipelineRun): string => {
+  const startTime = _.get(run, ['status', 'startTime'], null);
+  const completionTime = _.get(run, ['status', 'completionTime'], null);
+
+  // Duration cannot be computed if start time is missing or a completed/failed pipeline has no end time
+  if (!startTime || (!completionTime && pipelineRunStatus(run) !== 'Running')) {
+    return '-';
+  }
+  // Start Time is converted to UTC for correct comparison
+  const start = convertUTC(startTime).getTime();
```

Comment:
`getTime` is always in UTC.
This current conversion is odd and doesn't do anything different than just using `getTime`.
The question is whether the timestamps returned are in UTC. If yes, then the original implementation you had would work.

Reply:
I shouldn't have said anything earlier and looked at the unit tests timestamp value which shows the pipeline timestamps are in UTC already. I just confused you.

Reply:
reverted to original PR